components/button/index.tsx

```diff
@@ -3,7 +3,15 @@
 
 import s from './style.module.css'
 
-export default function Button({ children, href }) {
+export interface ButtonProps {
+  href: string
+  children: React.ReactChildren
+}
+
+export default function Button({
+  children,
+  href,
+}: ButtonProps): React.ReactElement {
```

Comment:
I think ideally if we're typing a component that can accept `children` we look to leverage the `React.FC` helper type. We'd have to change the definition to `const`:

```tsx
interface ButtonProps {
  href: string
}

const Button: React.FC<ButtonProps> = ({ children, href }) => { ... }
```

Reply:
What is the advantage of doing this?